apps/mull-ui/src/app/components/custom-file-upload/custom-file-upload.scss

```
@@ -0,0 +1,54 @@
+@import 'variables';
+
+.imageURL {
+  margin: 1.8em 5em;
```

```suggestion
  margin: 1.8rem 5rem;
```

Please also remove other unnecessary uses of `em` in `create-event.scss`.

---

I'd prefer to have another task where I clean up the UI and CSS.  Let me know what you think and I'll add it to the backlog if you agree.

---

yup, its better as a seperate bug or fix pr